src/pages/Student/StudentSession.jsx

```diff
@@ -0,0 +1,213 @@
+        <Typography
+          sx={{
+            backgroundColor: "#055c1c",
+            borderRadius: 2,
+            padding: 2,
+            margin: 1,
+            textAlign: "center",
+            fontWeight: "bold",
+            fontSize: 25,
+          }}
+        >
+          My Current Status: Confirm
+        </Typography>
+      </Box>
+      <Typography
+        sx={{
+          backgroundColor: "#ffffff",
+          borderRadius: 2,
+          padding: 1,
+          margin: 1,
+          textAlign: "center",
+          fontSize: 20,
+        }}
+      >
+        Start: {currentSession?.start}
+      </Typography>
+      <Typography
+        sx={{
+          backgroundColor: "#ffffff",
+          borderRadius: 2,
+          padding: 1,
+          margin: 1,
+          textAlign: "center",
+          fontSize: 20,
+        }}
+      >
+        End: {currentSession?.end}
+      </Typography>
+      <List>
+        {currentSession?.participant?.length > 0 ? (
+          currentSession?.participant.map((p) => (
+            <Card
+              key={p._id}
+              sx={{
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center",
+              }}
+            >
+              <Box>
+                <CardContent>
+                  <Typography variant="h5">{`Student: ${p.user.userInfo.name}`}</Typography>
+                </CardContent>
+              </Box>
+              <Box>
+                <CardContent>
+                  <Typography>{`Status: ${p.user.userStatus}`}</Typography>
+                </CardContent>
+              </Box>
+            </Card>
+          ))
+        ) : (
+          <Typography
+            sx={{
+              backgroundColor: "#f2f2f2",
+              padding: 2,
+              borderRadius: 2,
+              textAlign: "center",
+            }}
+          >
+            No sessions scheduled for this week
+          </Typography>
+        )}
+      </List>
+      <Typography
+        component={"h1"}
+        sx={{
+          backgroundColor: "#C84B31",
+          borderRadius: 2,
+          padding: 2,
+          margin: 1,
+          textAlign: "center",
+          fontWeight: "bold",
+          fontSize: 25,
+        }}
+      >
+        Discussion
+      </Typography>
+      <Box
+        component={"form"}
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "center",
+          alignItems: "center",
+          width: "100%",
+        }}
+        autoComplete="off"
+        onSubmit={handleCommentSubmit}
+      >
+        <div className={styles.formContainer}>
+          <AuthFormControl width="75%">
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "column",
+                justifyContent: "center",
+              }}
+            >
+              <br></br>
+            </Box>
+            <FormTextField
```

Comment:
If the FormTextField component doesn't require any regex to validate the text written, the props onHandleError value should be ()=>{}. (I have a textfield like this on the login page (password) ).